test/core/commands/codeBlockExpectedResult.txt

```diff
@@ -1 +1 @@
-{"Code":"A = B;\nC=1;", "LineIndices": [0, 1],"InPorts": ["B"], "OutPorts": ["A","C"], "Data": "null"}
\ No newline at end of file
+{"Code":"A = B;\nC=1;", "LineIndices": [0, 1],"InPorts": ["B"], "OutPorts": ["A","C"]}
```

Comment:
To pass `CanSendCodeBlockNodeUpdate()` test case correctly it requires changes from [this PR](https://github.com/Benglin/Dynamo/pull/256) where `Data` is not being included anymore